requirements.txt

```diff
@@ -0,0 +1,5 @@
+Flask==1.0.2
```

Comment:
You have `requirements.txt` and `backend/requirements.txt`. The first one should be removed as contains the same requirements as the later one and is outside `backend` directory.